ADOPTERS.md

```diff
@@ -1,3 +1,4 @@
-| Organization | Contact | Description of Use |
-| ------------ | ------- | ------------------ |
-| [Spotify](https://www.spotify.com) |[@alund](https://github.com/alund)| Main interface towards all of Spotify's infrastructure and technical documentation.|
+| Organization                       | Contact                                    | Description of Use                                                                  |
+| ---------------------------------- | ------------------------------------------ | ----------------------------------------------------------------------------------- |
+| [Spotify](https://www.spotify.com) | [@alund](https://github.com/alund)         | Main interface towards all of Spotify's infrastructure and technical documentation. |
+| [DFDS](https://www.dfds.com)       | [@carlsendk](https://github.com/carlsendk) | Internal self-service at DFDS.                                                      |
```

Comment:
Please make this change in a separate PR to not mix up discussions.

Reply:
Reverted :) We will re-introduce it in a later PR, we are just very eager to show our support for this great idea ;)